src/focus.service.ts

```diff
@@ -509,6 +509,22 @@ export class FocusService {
   private isFocusable(el: HTMLElement) {
     const role = el.getAttribute('role');
     const tabIndex = el.tabIndex;
+    let visible = el.offsetParent !== null;
+
+    for (let parent = el; parent !== this.root; parent = parent.parentElement) {
+      if (parent.classList.contains('hidden')) {
```

Comment:
Well but none of the other options work in our specific case. I mean if you comment out the class checking, there is no way that I've currently seen that makes the hidden tabs in frontend2 unselectable. If you have any suggestions that work (such as revamping the way the tabs are hidden in frontend2) please do tell